starter-code/pizza.js

```diff
@@ -1 +1,78 @@
 // Write your Pizza Builder JavaScript in this file.
+
+// to make a tomato sauce as default, toggle it first to OFF:
+//to make crust to be dafault
+$('.sauce').toggleClass('sauce-white');
+$('.crust').toggleClass('crust-gluten-free');
+$('.price-wsauce').toggle();
+$('.price-glut').toggle();
+
+//price of a plain pizza, displayed in #sum:
+var sum = 13;
+$('#sum').html(sum);
+
+//pepperoni toggel 
+//$(".selector").toggleClass(className)
+$('.btn-pepperonni').click(function(){
+  $('.pep').toggle();
+  $(this).toggleClass("active");
+  $('.price-pep').toggle();
+  if ($('.btn-pepperonni').hasClass('active')) {
+    sum += 1;
+  } else {
+    sum -=1;
+  }
+})
+
+//mushroom toggel 
+$('.btn-mushrooms').click(function(){
+  $('.mushroom').toggle();
+  $(this).toggleClass("active");
+  $('.price-mush').toggle();
+  if ($('.btn-mushrooms').hasClass('active')) {
+    sum += 1;
+  } else {
+    sum -=1;
+  }
+})
+
```

Comment:
Perhaps it might be useful here to create two functions that could help you avoid duplicate code:
e.g. `calculatePrice()` and `toggleIngredient()` for example.  A function like `setupDefaultToppings()` might also be handy in terms of readability.  Then the challenge is, how can you define and organize your classes to be able to do these things for every ingredient with one function?